homeassistant/components/modbus/const.py

```diff
@@ -12,6 +12,8 @@
 CONF_CURRENT_TEMP_REGISTER_TYPE = "current_temp_register_type"
 CONF_DATA_COUNT = "data_count"
 CONF_DATA_TYPE = "data_type"
+CONF_FAN = "fan"
```

Comment:
It would be better to import the fan integration domain and use when loading the platforms.